app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java

```diff
@@ -68,6 +70,7 @@ private void startQuizActivity(String quizID) {
 
         public CustomAdapter(Context context, ArrayList<Map.Entry<String, String>> entries) {
             this.entries = entries;
+            Collections.sort(this.entries, CustomComparator);
```

Comment:
@cjordan7 as you handled this one your PR I should probably remove all changes to HomeQuizListFragment, do you agree?

Reply:
Yes

Reply:
I removed the chages